Iniciante/2203/README.md

```diff
@@ -0,0 +1,91 @@
+# Problema:
+
+Fiddlesticks é um campeão do jogo League of Legends e tem como sua habilidade ultimate a "Tempestade de Corvos", ela funciona da seguinte maneira:
+
+Primeiro Fiddlesticks escolhe um local estratégico e prontamente ele se prepara para ressurgir em uma direção até uma certa distância, então ele se enraiza e canaliza a ultimate por exatamente 1.5 segundos, após esse tempo ele ressurge imediatamente no local alvo com uma revoada de corvos voando ao seu redor e causando muito dano.
+
+Fiddlesticks quer sua ajuda para saber se de uma certa posição é possível atingir um invasor com sua habilidade ultimate.
+
+Obs: Considere que Fiddlesticks sempre luta exatamente na direção do invasor e o invasor sempre tenta fugir na direção contrária a Fiddlesticks, em velocidade constante.
+
+<img src="https://resources.urionlinejudge.com.br/gallery/images/problems/UOJ_2203.png" />
+
+
+##### Link do problema: https://www.urionlinejudge.com.br/judge/pt/problems/view/2176
+ 
+ 
+# Resolução:
+
+O exercício possui alguns elementos de física e plano cartersiano e, apesar de apresentar muitas variáveis, não é tão complexo de se resolver. Primeiramente, vamos analisar o que é dado:
+- Posição (X,Y) do campeão Fiddlesticks;
+- Posição (X,Y) do inimigo;
+- Velocidade do inimigo fugindo;
+- Raio de alcance que o campeão pode realizar o ataque;
+- Raio do ataque em si;
+- O ataque leva 1.5 segundos para ser realizado (por isso a velocidade).
+
+Para resolver o problema, precisamos:
+- Calcular a distância atual entre os campeões;
+- Somar a distância que o inimigo se afasta em 1.5 segundos;
+- Verificar se essa distância está dentro dos raios de ataque.
+
+Começamos declarando as variáveis necessárias para o algoritmo ─ as entradas que o exercício fornece, do tipo inteiro, e as distâncias que vamos calcular, do tipo `double`.  
+Note que (Xf,Yf) são coordenadas de Fiddlesticks e (Xi,Yi) são coordenadas do inimigo.
+```c
+int Xf, Yf, Xi, Yi, V, R1, R2;
+double distancia, distX, distY;
+```
+
+O enunciado não deixa claro, mas devemos ler as entradas do exercício até que haja o fim de arquivo, ou seja, até o `EOF`. Para isso, colocamos a função `scanf()` dentro de uma estrutura `while()` e repetimos enquanto a entrada for diferente de `EOF`.
+
+```c
+while (scanf("%d %d %d %d %d %d %d", &Xf, &Yf, &Xi, &Yi, &V, &R1, &R2) != EOF) {
+
+    /* calculos de distancia e exibicao na tela */
+
+}
+```
+###### Sim, essa linha assusta, mas estamos apenas lendo dados (:
+
+Para calcular a distância entre os dois campeões, vamos utilizar funções de potência e raiz que podem ser revisadas [aqui](https://brasilescola.uol.com.br/matematica/distancia-entre-dois-pontos.htm). Essas funções ─ `pow()` e `sqrt()` ─ estão presentes na biblioteca `math.h`, importada no início do código.  
```

Comment:
```suggestion
Para calcular a [distância](https://brasilescola.uol.com.br/matematica/distancia-entre-dois-pontos.htm) entre os dois campeões, vamos utilizar funções de potência e raiz ─ `pow()` e `sqrt()` ─  presentes na biblioteca `math.h`, importada no início do código.  
```

Reply:
Aqui alterei onde fica o link do cálculo das distâncias pq da outra forma ficou parecendo um pouco que servia para revisar sobre funções de potência e raiz.
Fiquei na dúvida se esse `math.h` não deveria vir no início da resolução